scripts/utils/report_utils.py

```diff
@@ -0,0 +1,223 @@
+import os
+import re
+
+from .dbt_utils import get_deployment_version
+from .file_utils import ensure_directory_exists, read_file, write_file
+
+SCHEMA = "reporting"
+ROLE = "reporting"
```

Comment:
Based on recent [Slab doc](https://beyond-essential.slab.com/posts/deploy-reporting-schema-1c8h5col)
```suggestion
ROLE = "tamanu_reporting_user"
```